dali_tf_plugin/dali_tf_plugin_install_tool.py

```diff
@@ -128,9 +128,9 @@ def install(self):
         dali_cflags, dali_lflags = get_dali_build_flags()
         tf_cflags, tf_lflags = get_tf_build_flags()
         cuda_cflags, cuda_lflags = get_cuda_build_flags()
-        plugin_src = self.src_path + '/daliop.cc'
+        plugin_src = self.src_path + '/daliop.cc' + ' ' + self.src_path + '/dali_dataset_op.cc'
```

Comment:
maybe create a list of filenames and then construct this in a more generic way:
```for filename in filenames:
   plugin_src = plugin_src + ' ' + self.src_path + '/' + filename
```

Reply:
Done